edweiss-app/__test__/memento/card.test.tsx

```diff
@@ -13,6 +23,11 @@ import React from 'react';
 import { Button } from 'react-native';
 import { State } from 'react-native-gesture-handler';
 
+// ------------------------------------------------------------
+// ---------------------  Mock Data & Setup -------------------
```

Comment:
However those three lines could be useful since the structures of the mocks could make the code less readable.